opm/autodiff/StandardWell_impl.hpp

```diff
@@ -303,7 +304,8 @@ namespace Opm
     wellVolumeFractionScaled(const int compIdx) const
     {
 
-        const double scal = scalingFactor(compIdx);
+        const int legacyCompIdx = ebosCompIdxToFlowCompIdx(compIdx);
+        const double scal = scalingFactor(legacyCompIdx);
```

Comment:
Same question apply here: why is it changed?

Reply:
answered above